### ISSUE TITLE
An agenda has been installed as an ICE

### ISSUE THREAD

**Issue by JeffSkyrunner** [dagguh/netrunner#106]
**Describe the bug**
Maybe it is meant at the current state, but sometimes an agenda is set unrezzed in an "ICE" position, protecting an asset

**To Reproduce**
Simply play

**Expected behavior**
Only ICE should be on ICE position

**Comment by dagguh** [comment]
Wow! Installing ICE is not implemented at all. So you just let the corp play, and it will install something in the ICE position? Could you attach a screenshot?

**Comment by dagguh** [comment]
I think I know:
<img width="805" alt="image" src="https://user-images.githubusercontent.com/1194704/82382082-a6be9300-9a2b-11ea-9627-4bb4f1c2ab26.png">
When you run this, you access an agenda.
Corp overinstalls the agenda in a remote, so it lands in Archives 😁.
So the problem is: it's unclear which servers are which.

**Comment by JeffSkyrunner** [comment]
Exactly :)

**Comment by JeffSkyrunner** [comment]
Suggestion: we could create/use some specific image to use for R&D and Archives, while we can use the Corp identity for HQ, like in all cards game.

**Comment by dagguh** [comment]
Agreed. The servers are bigger than the cards their contain. There's room for a distinct border and an area for icons etc.

**Comment by JeffSkyrunner** [comment]
I think we can close this, I have never seen this happening again

**Comment by dagguh** [comment]
I feel it's still not obvious. Especially with empty Archives. Each server should have a background/edge (maybe with distinct color) and a label/icon representing RD/HQ/Archives/Remote
